Gems/OpenXRVk/Code/Source/Platform/Common/Default/InputDeviceXRController_Default.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <OpenXRVk/InputDeviceXRController.h>
+#include <../Common/Default/OculusTouch_Default.h>
```

Comment:
[suggestion] `#include <../Common/Default/OculusTouch_Default.h>` -> `#include "OculusTouch_Default.h"` since they are in the same folder. Normally I incline towards not using quotes, but in this case seems reasonable if it's not trivial to add `/Common/Default/` to the include path for PAL reasons.

Reply:
Thanks for the review!  I agree with you, to prefer `<>` in most cases.  Having to `..` up a directory is not ideal, but as I mentioned before in PAL we commonly break certain conventions.  Afaik, we don't add Common/Default etc folders to CMake include paths, because you'd have to potentially change that per platform and it gets messy.  So in this case I think doing `""` here is an OK change to make.